app/controllers/groups_controller.rb

```diff
@@ -68,53 +63,13 @@ def new
 
   def create
     @group = group_params
-
     @is_system_group = params[:system_group]
-    @management_group_concept_id = @group.delete('initial_management_group')
-
     @group['provider_id'] = current_user.provider_id unless @is_system_group
 
     group_creation_response = cmr_client.create_group(@group, token)
 
     if group_creation_response.success?
-      concept_id = group_creation_response.body['concept_id']
-
-      single_instance_identity_object = construct_permission_object(
-                                          group_id: @management_group_concept_id,
-                                          permissions: SingleInstanceIdentityPermissionsHelper::GROUP_MANAGEMENT_PERMISSIONS,
-                                          target: 'GROUP_MANAGEMENT',
-                                          identity_type: 'single_instance_identity',
-                                          target_id: concept_id
-                                        )
-
-      management_group_response = cmr_client.add_group_permissions(single_instance_identity_object, token)
-
-      if management_group_response.success?
-        Rails.logger.info("Single Instance Identity ACL for #{@management_group_concept_id} to manage #{concept_id} successfully created by #{current_user.urs_uid}")
-      else
-        Rails.logger.error("Create Single Instance Identity ACL error: #{management_group_response.inspect}")
-
-        delete_group_response = cmr_client.delete_group(concept_id, token)
-
-        if delete_group_response.success?
-          # form data is still available to populate the form
-          set_previously_selected_members(group_params.fetch('members', []))
-          flash[:error] = 'There was an issue creating the group. Please try again.'
-
-          render :new and return
-        else
-          # Just successfully created a group, but failed on creating a single
-          # instance identity (group management) acl, and then failed on deleting
-          # the group
-          Rails.logger.error("Group Deletion Error: #{delete_group_response.inspect}")
-          flash[:error] = "Group [#{@group['name']}] was created but there were issues with the Initial Management Group permissions. Please delete this group and try again."
-
-          redirect_to group_path(concept_id) and return
-        end
-      end
-      flash[:success] = 'Group was successfully created.'
-
-      redirect_to group_path(concept_id)
+      redirect_to group_path(group_creation_response.body.fetch('concept_id', nil)), flash: { success: 'Group was successfully created.' }
```

Comment:
`group_creation_response.body.fetch('concept_id', nil)` is the same thing that you had before `group_creation_response.body['concept_id']`